app/assets/javascripts/map_display.js.coffee

```diff
@@ -71,11 +71,17 @@ class window.LeafletMap
     return
 
   addLayers: (opts = {}) ->
-    opacity = opts.opacity || 0.8
     baseLayers = {}
     for tileServer, idx in $("#map-tiles").data("tileservers")
       continue if (tileServer.url == "" or tileServer.name == "")
-      tileLayer = L.tileLayer(tileServer.url)
+      tileLayer = if tileServer.type == "wms"
+         tileLayer = L.tileLayer.wms(tileServer.url, {
+            'layers': '0',
+            'format': 'image/jpeg',
+            'srs': "EPSG:3857"
```

Comment:
srs?  Surely its [crs](http://leafletjs.com/reference-0.7.7.html#tilelayer-wms-crs).


It makes me think we should just have options as a hash as the layer is likely to be different.  What do you think about having a Postgres JSON type as `tile_server1_options` and passing it in here?  Then we can just leave that to the admin user, what do you think?